ksqldb-common/src/main/java/io/confluent/ksql/function/UdfFactory.java

```diff
@@ -79,6 +80,10 @@ public String toString() {
         + '}';
   }
 
+  public synchronized KsqlScalarFunction getUdfFunction(final List<SqlArgument> argTypes) {
```

Comment:
Added another `getFunction` here instead of modifying the original because table functions still use the other `getFunction` method